app/routes/maps/create/index.tsx

```diff
@@ -502,7 +502,8 @@ function RemoveLayerModal({ url, show, setShow }: RemoveLayerModalProps) {
                 : "Are you sure you want to remove all layers?"}
             </h3>
             <div className="flex justify-center gap-4">
-              <Button color="failure" type="submit" name="url" value={url}>
+              {/* Using !text-white to override Flowbite React 0.4.9 styling issue in light mode */}
```

Comment:
I don't think this is the proper way to handle this. Check tailwind config to modify what the button should look like when it's not on hover. Check "color=failure".